homeassistant/components/fan/__init__.py

```diff
@@ -234,10 +234,10 @@ class FanEntity(ToggleEntity, cached_properties=CACHED_PROPERTIES_WITH_ATTR_):
     entity_description: FanEntityDescription
     _attr_current_direction: str | None = None
     _attr_oscillating: bool | None = None
-    _attr_percentage: int | None
+    _attr_percentage: int | None = 0
     _attr_preset_mode: str | None = None
     _attr_preset_modes: list[str] | None = None
-    _attr_speed_count: int
+    _attr_speed_count: int = 100
```

Comment:
I kept them out of #130930 because the default is not `None` and I wanted time to think more about it...
I now think this change also makes sense.